examples/datasets/plot_limo_data.py

```diff
@@ -236,63 +215,52 @@
     plot_compare_evokeds(evokeds,
                          axes=ax,
                          ylim=dict(eeg=(-20, 20)),
-                         colors=colors,
-                         split_legend=True,
                          picks=electrode,
-                         cmap=(name + " Percentile", "magma"))
-    fig.show()
+                         cmap=("Standardized phase coherence", "magma"))
 
 ###############################################################################
 # As shown above, there are some considerable differences between the
-# activation patterns evoked by stimuli with low vs. high phase-coherence.
+# activation patterns evoked by stimuli with low vs. high phase-coherence at
+# the chosen electrodes.
+#
 #
-# In particular, differences appear to be most pronounced along fronto-central
-# and occipital electrodes.
```

Comment:
you've only shown 2 electrodes, I don't think it's fair to generalize to whole regions of electrodes from that.

Reply:
you're right. I think I was already thinking about the results shown futher down. Thanks for pointing this out.